### PR TITLE
SAK-42545 Only load rubric association js when needed

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

**Comment by mpellicer** [comment]
Awesome! good idea.

However, I see changes from #7298, am I right?

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js [hunk 1]
```diff
@@ -70,9 +70,11 @@ export class SakaiRubricCriteria extends RubricsElement {
                   ${r.title}
                   <sakai-rubric-criterion-rating-edit criterion-id="${c.id}" @save-rating="${this.saveRating}" @delete-rating="${this.deleteRating}" minpoints="${c.pointrange ? c.pointrange.low : 0}" maxpoints="${c.pointrange ? c.pointrange.high : 0}" rating="${JSON.stringify(r)}"></sakai-rubric-criterion-rating-edit>
                 </h5>
-                <p>
+                <div class="div-description">
```
Comment (mpellicer) [inline rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js]: Is this change inherited from #7298 ?

File: msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/dfAllMessages.jsp [hunk 2]
```diff
@@ -14,15 +14,16 @@
 
 <!-- messageforums-app/src/webapp/jsp/discussionForum/message-->
 
-<script type="text/javascript">includeLatestJQuery("msgcntr");</script>
```
Comment (mpellicer) [inline msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/dfAllMessages.jsp]: Are all of these changes related to the PR?

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js (hunk 1 above)
Comment (adrianfish) [reply rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js]: Shouldn't be. I based off master. You have me worried though.

File: msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/dfAllMessages.jsp (hunk 2 above)
Comment (adrianfish) [reply msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/dfAllMessages.jsp]: I took the type attribute off the js scripts. Is that what you mean? They're not needed and I like to clean them up as I go.

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js (hunk 1 above)
Comment (adrianfish) [reply rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js]: Good job you spotted that. I must have copied some code from the deployed webapp. Nice one!

**Pull request merged**